src/engine/Actor.ts

```diff
@@ -480,11 +480,9 @@ export class ActorImpl extends Class implements IActionable, IEvented, IPointerE
       this.actionQueue = new ActionQueue(this);
       this.actions = new ActionContext(this);
 
-      // default anchor is in the middle
-      this.anchor = new Vector(.5, .5);
-
       // Initialize default collision area to be box
       this.body.useBoxCollision();
+      this._initDefaults();
```

Comment:
You will also need to move the `_initDefaults()` at least for `anchor` above `this.body.useBoxCollision()`;

`useBoxCollision()` needs the anchor to perform calculations for physics geometry

```typescript
this._initDefaults();
// Initialize default collision area to be box
this.body.useBoxCollision();
```